code/game/dna/dna_modifier.dm

```diff
@@ -53,6 +53,7 @@
 	var/mob/living/carbon/occupant = null
 	var/obj/item/weapon/reagent_containers/glass/beaker = null
 	var/opened = 0
+	var/allowed_braintype = /obj/item/organ/internal/brain/slime //Incase an admin wishes to varchange this to allow another type of brain, or subtypes are needed.
```

Comment:
Maybe toss this var on the brain instead, so if admins specifically want something to be brain-cloneable they don't have to touch the machine with pathing?